chapters/03-funcoes.md

```diff
@@ -45,7 +45,7 @@ console.log(power(2, 10));
 // → 1024
 ```
 
-Algumas funções produzem um valor, como as funções `power` e `square` vistas acima, e outras não, como no exemplo de `makeNoise`, que produz apenas um “efeito colateral”. A declaração `return` é usada para determinar o valor de retorno da função. Quando o controle de execução interpreta essa declaração, ele sai imediatamente do contexto da função atual e disponibiliza o valor retornado para o código que invocou a função. A palavra-chave `return` sem uma expressão após a mesma irá fazer com que a função retorne  `undefined`.
+Algumas funções produzem um valor, como as funções `power` e `square`  acima, e outras não, como no exemplo de `makeNoise`, que produz apenas um “efeito colateral”. A declaração `return` é usada para determinar o valor de retorno da função. Quando o controle de execução interpreta essa declaração, ele sai imediatamente do contexto da função atual e disponibiliza o valor retornado para o código que invocou a função. A palavra-chave `return` sem uma expressão após, faz com que o retorno seja  `undefined`.
```

Comment:
:-1: no trecho `[...] funções power e square acima`.  Foi acrescentado um espaço extra.  Tirando isso, está ok.

:-1: no trecho `[...] sem uma expressão após, faz com que o retorno seja  undefined.`  Basta verificar o trecho original: `The return keyword without an expression after it will cause the function to return undefined.`  Eu concordaria com alterações se fossem ajudar (simplificando, detalhando, dando ênfase, etc...) no entendimento e nesse caso não vejo vantagens na alteração, acho que ser menos específico nesse caso pode causar até mais confusão por parte do leitor.

Além disso, há um espaço extra antes de `undefined` que passou despercebido na revisão e na super.